### PR TITLE
Business added Resume

### PR DESCRIPTION
**Pull request by barys21**

### CODE REVIEW

File: UtopiaCity/Controllers/Business/ResumeController.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        [HttpGet]
+        public async Task<IActionResult> Edit(string id)
+        {
+            if (string.IsNullOrWhiteSpace(id))
+            {
+                return NotFound();
+            }
+
+            var resume = await _resumeAppService.GetById(id);
+            ViewData["ResidentAccountId"] = new SelectList(await _resumeAppService.GetAllResidentAccount(), "Id", "FIO", resume.ResidentAccountId);
```
Comment (crocomoth) [inline UtopiaCity/Controllers/Business/ResumeController.cs:55]: Probably can be moved to separate method. Also constants can be used

File: UtopiaCity/Data/ApplicationDbContext.cs [hunk 2]
```diff
@@ -40,6 +40,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         public DbSet<PermitedModel> PermitedModel { get; set; }
         public DbSet<Ticket> Tickets { get; set; }
         public DbSet<Airline> Airlines { get; set; }
+
+        public DbSet<ResidentAccount> RersidentAccount { get; set; }
```
Comment (crocomoth) [inline UtopiaCity/Data/ApplicationDbContext.cs:63]: empty lines not needed

File: UtopiaCity/Data/Migrations/ApplicationDbContextModelSnapshot.cs [hunk 3]
```diff
@@ -327,9 +327,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<DateTime>("Days")
                         .HasColumnType("datetime2");
 
-                    b.Property<string>("FlightWeather")
```
Comment (crocomoth) [inline UtopiaCity/Data/Migrations/ApplicationDbContextModelSnapshot.cs:403]: ?

File: UtopiaCity/Models/Business/Resume.cs [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    public class Resume
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public string Id { get; set; }
+
+        public string ResidentAccountId { get; set; }
+
+        public ResidentAccount ResidentAccount { get; set; }
+
+        public string ProfessionId { get; set; }
```
Comment (crocomoth) [inline UtopiaCity/Models/Business/Resume.cs:16]: empty line?

**Review by crocomoth — CHANGES_REQUESTED** [review]

File: UtopiaCity/Data/Migrations/ApplicationDbContextModelSnapshot.cs (hunk 3 above)
Comment (barys21) [reply UtopiaCity/Data/Migrations/ApplicationDbContextModelSnapshot.cs:403]: I deleted content in snapshot and re-filled, then it went like that

File: UtopiaCity/Models/Business/Resume.cs (hunk 4 above)
Comment (barys21) [reply UtopiaCity/Models/Business/Resume.cs:16]: deleted

**Review by crocomoth — APPROVED** [review]

**Review by IvanSultanov — APPROVED** [review]

**Review by AnatoliyDubovtsev — APPROVED** [review]

File: UtopiaCity/Controllers/Business/ResumeController.cs (hunk 1 above)
Comment (barys21) [reply UtopiaCity/Controllers/Business/ResumeController.cs:55]: I'll leave it like that for now

**Pull request merged**